civihr_employee_portal/css/custom.css

```diff
@@ -318,6 +318,14 @@ span.appraisals-employee-legend {
   padding: 0;
 }
 
+#reportPivotTable [uib-datepicker-popup-wrap] table {
+  margin: 0;
+}
+
+#reportPivotTable table.uib-daypicker tbody td {
+  padding: 7px 0;
```

Comment:
Does this padding matched the padding in mockups?

Reply:
This padding is not part of the mocks. It was added to override a padding added by SSP Theme [here](https://github.com/compucorp/civihr-employee-portal-theme/blob/staging/civihr_default_theme/scss/layout/_table.scss#L11). That padding is useful for the report, fields, etc. but not for the date picker.

The value of `7px 0` came for the default value the date picker would have taken if not for the rule stated above.

<table>
  <tr>
    <th>Before</th>
    <th>After</th>
  </tr>
    <td><img src="https://user-images.githubusercontent.com/1642119/34526821-ff4069e4-f079-11e7-8d86-2d2e7e5459d1.png" /></td>
    <td><img src="https://user-images.githubusercontent.com/1642119/34526710-afc9e34a-f079-11e7-993c-e259fd4b7b19.png" /></td>
</table>